.semaphore/semaphore.yml

```diff
@@ -92,9 +92,9 @@ blocks:
   dependencies: []
   task:
     jobs:
-    - name: "Prerequisites"
+    - name: "Check generated files"
       commands:
-      - make gen-semaphore-yaml
```

Comment:
Do we run `gen-semaphore-yaml` elsewhere?

Reply:
It's included as part of `generate` now